examples/plot_hf_hub.py

```diff
@@ -89,7 +90,8 @@
 
 # you can put your own token here, or set it as an environment variable before
 # running this script.
-token = os.environ["HF_HUB_TOKEN"]
+# token = os.environ["HF_HUB_TOKEN"]
+token = "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"
```

Comment:
Is this exposing a prod token?

Reply:
Much better with environment variable! Btw you can refresh a given token to expire this one